### PR TITLE
Fixed authorizations api

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: lib/Github/Api/Authorizations.php [hunk 1]
```diff
@@ -40,6 +40,16 @@ public function remove($id)
 
     public function check($id, $token)
```
Comment (stof) [inline lib/Github/Api/Authorizations.php]: I suggest renaming ``id`` to ``clientId`` to be more explicit about which id it is (the github doc describes it as ``client_id`` btw): https://developer.github.com/v3/oauth_authorizations/#check-an-authorization

**Comment by stof** [comment]
Resetting an authorization is also missing: https://developer.github.com/v3/oauth_authorizations/#reset-an-authorization

File: lib/Github/Api/Authorizations.php (hunk 1 above)
Comment (pilot) [inline lib/Github/Api/Authorizations.php]: @GrahamCampbell rename please it would be clear

**Comment by cursedcoder** [comment]
done by #277

**Pull request closed without merging**